### PR TITLE
[OSDOCS-5012]: Full Azure CPMS support

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.13+

Issue:
[OCPCLOUD-1764](https://issues.redhat.com//browse/OCPCLOUD-1764) | [OSDOCS-5012](https://issues.redhat.com//browse/OSDOCS-5012)

Link to docs preview:
[Supported cloud providers](https://55407--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-getting-started.html#cpmso-platform-matrix_cpmso-getting-started)

QE review:
- [x] QE has approved this change.

Additional information:
* Rel notes: #55409
* Will squash commits after all reviews are in.

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@odvarkadaniel this PR is ready for dev review, PTAL

File: machine_management/control_plane_machine_management/cpmso-getting-started.adoc [hunk 1]
```diff
@@ -51,7 +51,9 @@ The status of the Operator after installation depends on your cloud provider and
 |====
 [.small]
 --
-1. AWS clusters that are upgraded to version 4.12 from an earlier version require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+1. AWS clusters that are upgraded from version 4.11 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+2. For Azure clusters that are upgraded to version 4.13 from an earlier version, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[create and activate a CR].
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply machine_management/control_plane_machine_management/cpmso-getting-started.adoc:55]: I misunderstood the state of a cluster that is updated to 4.13. I will revise this accordingly

File: machine_management/control_plane_machine_management/cpmso-getting-started.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ In {product-title} {product-version}, the Control Plane Machine Set Operator is
 
 The status of the Operator after installation depends on your cloud provider and the version of {product-title} that you installed on your cluster.
 
-.Control Plane Machine Set Operator implementation matrix
+.Control plane machine set implementation for {product-title} {product-version}
```
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-getting-started.adoc:32]: Revised table to show only 4.13 status, and added `{product-title} {product-version}` to hopefully make that clear. @odvarkadaniel what do you think, better now?

Comment (odvarkadaniel) [reply machine_management/control_plane_machine_management/cpmso-getting-started.adoc:32]: LGTM!

**Comment by jeana-redhat** [comment]
@sunzhaohua2 PTAL

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done
/LGTM

@jeana-redhat Remember to squash the commits, otherwise you are good!

File: machine_management/control_plane_machine_management/cpmso-getting-started.adoc [hunk 3]
```diff
@@ -51,7 +51,8 @@ The status of the Operator after installation depends on your cloud provider and
 |====
 [.small]
 --
-1. AWS clusters that are upgraded to version 4.12 from an earlier version require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+1. AWS clusters that are upgraded from version 4.11 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+2. Azure clusters that are upgraded from version 4.12 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
```
Comment (jeana-redhat) [inline machine_management/control_plane_machine_management/cpmso-getting-started.adoc:55]: @sunzhaohua2 I updated this based on [your comment](https://github.com/openshift/openshift-docs/pull/55409#discussion_r1123975817) on the release notes. The table is a little easier to look at in the preview [here](https://55407--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-getting-started.html#cpmso-platform-matrix_cpmso-getting-started) :) PTAL and lmk if this is accurate.

Comment (sunzhaohua2) [reply machine_management/control_plane_machine_management/cpmso-getting-started.adoc:55]: LGTM, thanks @jeana-redhat

**Pull request merged**